pkg/controllers/custompackage/controller_test.go

```diff
@@ -38,7 +44,7 @@ func TestReconcileCustomPkg(t *testing.T) {
 		ErrorIfCRDPathMissing: true,
 		Scheme:                s,
 		BinaryAssetsDirectory: filepath.Join("..", "..", "..", "bin", "k8s",
-			fmt.Sprintf("1.27.1-%s-%s", runtime.GOOS, runtime.GOARCH)),
+			fmt.Sprintf("1.29.1-%s-%s", runtime.GOOS, runtime.GOARCH)),
```

Comment:
Do you want to pull this out as a constant? It's used again below

Reply:
We need to make this more dynamic and define it at a project level. We need this in make file too. Probably better to just pull from a file or something similar. But I think it's out side scope for this PR.